packages/@aws-cdk/aws-efs/README.md

```diff
@@ -37,6 +37,27 @@ const fileSystem =  new FileSystem(this, 'EfsFileSystem', {
 });
 ```
 
+### Access Point
+
+An access point is an application-specific view into an EFS file system that applies an operating system user and
+group, and a file system path, to any file system request made through the access point. The operating system user
+and group override any identity information provided by the NFS client. The file system path is exposed as the
+access point's root directory. Applications using the access point can only access data in its own directory and
+below. To learn more, see [Mounting a File System Using EFS Access Points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html).
+
+Use `AccessPoint` to create an access point:
+
+```ts
+new AccessPoint(stack, 'AccessPoint', {
```

Comment:
> @rix0rrr I probably still didn't get it.
>
> Actually we will use `AccessPoint` in Lambda efs mount support https://github.com/aws/aws-cdk/pull/8602
> I am planing to pass the `AccessPoint` to the Lambda filesystemConfigs like this:
> 
> ```ts
>  filesystemConfigs: {
>     filesystem: lambda.LambdaFileSystem.fromEfsFileSystem(stack, accessPoint),
>     localMountPath: '/mnt/msg',
>   }
> ```
> 
> I also tried to implement `addAccessPoint` for `FileSystem` construct but this will create two `Resource`.

If it helps, @rix0rrr is suggesting to create something like this https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-cognito/lib/user-pool.ts#L613

We can also add this later if you prefer; it's a backwards compatible change.

Reply:
OK. I prefer to add this later because I still encounter a little issue by adding this. Will continue this in another PR.